### ISSUE TITLE
Double page support

### ISSUE THREAD

**Issue by B0pol** [g-fb/mangareader#2]
It would be nice to have double page support, be able to change whether to show first page alone first (and be able to set it in toolbar and keyboard shortcut).

**Comment by g-fb** [comment]
https://github.com/g-fb/mangareader#status

**Comment by groutoutlook** [comment]
hi @g-fb , may I know if you have come out of the maintanance status? I'm pretty hyped up after discovered and using this... and would love to send PR for this, since there are some features that I want to add to `mangareader`.

**Comment by g-fb** [comment]
> may I know if you have come out of the maintanance status?

Not really. I'm very rarely using it since I wrote a more advanced app with QML.

> would love to send PR for this

Double page support has been added.

> there are some features that I want to add to mangareader

You can submit PRs for new features, though i recommend you open an issue before to discuss the feature.
If you want to make many changes, you might want to fork.
